lib/elasticity/strategies/alias_index.rb

```diff
@@ -279,11 +288,20 @@ def mapping
 
       private
 
+      def build_index_name
+        ts = String.new
+        if @use_new_timestamp_format == true
+          ts = Time.now.utc.strftime("%Y%m%d%H%M%S%6N")
```

Comment:
Any chance we could add some delimiters to make this easier to parse by humans reading the index names?  Something like `2019_10_27_211410` (not sure if `_` is an acceptable delimiter, but whatever is could be useful here.

Reply:
yeah, we could.   but this format should be familiar to rails devs
https://github.com/doximity/doximity/tree/master/db/migrate  (which is why I picked it)

Reply:
and I was a bit scared of some future upgrade declaring other chars `off limits`

Reply:
Yeah - I was thinking that same thing about the chars too.  I'm good either way, just figured I'd mention it.

Reply:
👍 
fwiw,  most `human interaction` is usually via the aliases anyway